src/ImageSharp.Web/DependencyInjection/ImageSharpBuilderExtensions.cs

```diff
@@ -23,14 +22,14 @@ public static class ImageSharpBuilderExtensions
         /// <summary>
         /// Sets the given <see cref="IRequestParser"/> adding it to the service collection.
         /// </summary>
-        /// <typeparam name="TParser">The type of class implementing <see cref="IRequestParser"/>to add.</typeparam>
+        /// <typeparam name="TImplementation">The type of class implementing <see cref="IRequestParser"/> to add.</typeparam>
```

Comment:
`TParser` is better here. Even `T` is better than `TImplementation` which is wordy and doesn't add value.

Reply:
I hesitated between changing it to `T` or `TImplementation` because the latter is indeed a bit wordy and doesn't add the same value as when used in the service collection extension methods (that have `<TService, TImplementation>`).

I've mentioned my reasoning for this in the PR description and would actually prefer to change it to `T` instead, if you're okay with that:

> This PR also renames the generic type argument to TImplementation, because the some names were hard to distinguish with the interface name (e.g. TCacheHash and ICacheHash) and this aligns with the naming used on the service collection extensions.

Reply:
The existing naming is a convention we use throughout the Six Labors libraries e.g. TPixel and works well for us. It’s far more consistent than the .NET runtime and I’d like to keep it as is.